value.go

```diff
@@ -793,6 +793,10 @@ func (v Value) Clone() Value {
 }
 
 func makeInt96(bits []byte) (i96 deprecated.Int96) {
+	if cap(bits) < 12 {
```

Comment:
This should be `len(bits) < 12` I believe?

Reply:
it kinda depends on how we are going to treat slices that have offered less than 12 bytes, use their underlying array data or append 0 values? if the later is the choice, then we should use len and not cap.

Reply:
My understanding is that the original issue was that some parquet files may contain int96 values with less than 12 bytes in the "last value", so that would materialize as `len(bits) < 12`, or am I missing something?